extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -602,18 +608,25 @@ function processCheckboxField(context: FieldContext): void {
  * @param context The context to use to create the field
  */
 function processFilePickerField(context: FieldContext): FilePickerInputs {
+	const inputWidth = parseInt(context.fieldInfo.inputWidth!);
+	if (inputWidth === NaN) {
+		// this is a dev time only error
```

Comment:
This isn't really true, could happen in a stable build too just as easily if someone didn't test thoroughly.

Anyways I would suggest you look into getting an actual JSON schema set up for the resource deployment contribution as well so that these mistakes are easier to avoid. Right now we have a mix of straight int values and '##px' string values for inputWidth values so I think it'd be best to just be completely consistent here and switch everything over to int if you're going to be doing this type of logic. Or you could add logic to check for px and trim it off - whatever your preference is I guess.

Reply:
I am not planning to take a task of that size right now, so i am just going to remove the px from the width string for now.